From 693ed2d6f6fa51f0446a2c133cb61997cbd27414 Mon Sep 17 00:00:00 2001
From: Piyush Mehta <piyush.mehta@xilinx.com>
Date: Fri, 3 Apr 2020 18:41:52 +0530
Subject: [PATCH 1320/1851] usb: dwc3: otg: remove otg static export warning

commit 539671edfa356aa86706251b642f096bdef08396 from
https://github.com/Xilinx/linux-xlnx.git

This patch removes the static export warning,
when USB OTG feature enables.

Signed-off-by: Piyush Mehta <piyush.mehta@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/usb/dwc3/otg.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/usb/dwc3/otg.c b/drivers/usb/dwc3/otg.c
index bfa954b8aaa6..6807e5426a48 100644
--- a/drivers/usb/dwc3/otg.c
+++ b/drivers/usb/dwc3/otg.c
@@ -1720,7 +1720,6 @@ static void dwc3_otg_set_peripheral(struct usb_otg *_otg, int yes)
 
 	set_peri_mode(otg, yes);
 }
-EXPORT_SYMBOL(dwc3_otg_set_peripheral);
 
 static int dwc3_otg_set_periph(struct usb_otg *_otg, struct usb_gadget *gadget)
 {
@@ -1861,7 +1860,6 @@ static int otg_end_session(struct usb_otg *otg)
 {
 	return dwc3_otg_end_session(otg);
 }
-EXPORT_SYMBOL(otg_end_session);
 
 static int dwc3_otg_received_host_release(struct usb_otg *x)
 {
-- 
2.31.1

